examples/hikari.py

```diff
@@ -2,7 +2,7 @@
 
 import hikari
 from songbird import ytdl
-from songbird.hikari import Voicebox
+from songbird.integration.hikari import Voicebox
```

Comment:
This file structure shouldn't be changed. I know what my codebase looks like so I would prefer not to change it.

Reply:
You can make "stub" files, if you want

hikari.py:
```
from songbird.integration.hikari import *
```

Reply:
Support for older versions + soft transition in future